### DISCUSSION TITLE
Cubase

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#72] [General]
# Cubase

Check if Cubase is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/9553f22d-1b83-4ef5-af35-9ddec0c8b65d

**Comment by talynone** [comment]
Preview ARM native version is now available:

https://helpcenter.steinberg.de/hc/en-us/articles/21829527504530-Preview-versions-for-Windows-on-Arm-WoA
